docs/pages/configuration/pipelines/functions/run_dependency_pipelines.mdx

```diff
@@ -2,3 +2,37 @@
 title: Function run_dependency_pipelines
 sidebar_label: run_dependency_pipelines
 ---
+
+Runs dependencies for pipelines.
+
+## Arguments
+
+### `--pipeline`
+
+The pipeline to be deployed from the dependency.
```

Comment:
`deployed` is probably the wrong term here as it should be rather executed or run as we don't really know what functionality this pipeline actually fulfills

Reply:
I chose to use executed here.